py3status/core.py

```diff
@@ -302,7 +302,8 @@ def notify_user(self, msg, level='error'):
         if not dbus:
             msg = 'py3status: {}'.format(msg)
         if level != 'info':
-            msg += ' Please try to fix this and reload i3wm (Mod+Shift+R)'
+            fix_msg = '{} Please try to fix this and reload i3wm (Mod+Shift+R)'
```

Comment:
A compromise? (shorter message)
`msg = '{} Please fix and reload i3wm (Mod+Shift+R)'.format(msg) `